frontend/src/components/LowerButtonNavBar.js

```diff
@@ -3,27 +3,14 @@ import Box from '@mui/material/Box';
 import BottomNavigation from '@mui/material/BottomNavigation';
 import BottomNavigationAction from '@mui/material/BottomNavigationAction';
 import RestoreIcon from '@mui/icons-material/Restore';
-import FavoriteIcon from '@mui/icons-material/Favorite';
-import ArchiveIcon from '@mui/icons-material/Archive';
 import Paper from '@mui/material/Paper';
-
-function refreshMessages() {
-    const getRandomInt = (max) => Math.floor(Math.random() * Math.floor(max));
-
-    return Array.from(new Array(1)).map(
-        () => messageExamples[getRandomInt(messageExamples.length)],
-    );
-}
+import TrendingUpIcon from '@mui/icons-material/TrendingUp';
+import AccountBalanceWalletIcon from '@mui/icons-material/AccountBalanceWallet';
 
 export default function LowerButtonNavBar() {
+
     const [value, setValue] = React.useState(0);
     const ref = React.useRef(null);
-    const [messages, setMessages] = React.useState(() => refreshMessages());
-
-    React.useEffect(() => {
-        ref.current.ownerDocument.body.scrollTop = 0;
-        setMessages(refreshMessages());
-    }, [value, setMessages]);
 
     return (
         <Box sx={{ pb: 7 }} ref={ref}>
```

Comment:
Do you really need an ref here?

Reply:
We don't need, thanks for your hint. Meanwhile changed with us3.1.